frontend/src/scenes/toolbar-launch/AuthorizedUrls.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react'
+import './AuthorizedUrls.scss'
+import clsx from 'clsx'
+import { useActions, useValues } from 'kea'
+import { LemonTag } from 'lib/components/LemonTag/LemonTag'
+import { LemonButton } from 'lib/components/LemonButton'
+import { Input } from 'antd'
+import { authorizedUrlsLogic } from './authorizedUrlsLogic'
+import { isMobile } from 'lib/utils'
+import { LemonRow } from 'lib/components/LemonRow'
+import { IconDelete, IconEdit, IconOpenInApp, IconPlus } from 'lib/components/icons'
+import { Spinner } from 'lib/components/Spinner/Spinner'
+import { Form } from 'kea-forms'
+import { LemonInput } from 'lib/components/LemonInput/LemonInput'
+import { Field } from 'lib/forms/Field'
+import Typography from 'antd/lib/typography'
+
+interface AuthorizedUrlsTableInterface {
+    pageKey?: string
+    actionId?: number
+}
+
+function EmptyState({
+    numberOfResults,
+    isSearching,
+}: {
+    numberOfResults: number
+    isSearching: boolean
+}): JSX.Element | null {
+    if (numberOfResults > 0) {
+        return null
+    }
+
+    return isSearching ? (
+        <LemonRow outlined fullWidth size="large" className={clsx('AuthorizedUrlRow')}>
+            There are no authorized URLs that match your search.
+        </LemonRow>
+    ) : (
+        <LemonRow outlined fullWidth size="large" className={clsx('AuthorizedUrlRow')}>
+            There are no authorized URLs or domains. Add one to get started.
+        </LemonRow>
+    )
+}
+
+function AuthorizedUrlForm({ actionId }: { actionId?: number }): JSX.Element {
+    const logic = authorizedUrlsLogic({ actionId })
+    const { isProposedUrlSubmitting, proposedUrlHasErrors, urlToEdit } = useValues(logic)
+    const { cancelProposingUrl } = useActions(logic)
+    return (
+        <Form
+            logic={authorizedUrlsLogic}
+            props={{ actionId }}
+            formKey="proposedUrl"
+            enableFormOnSubmit
+            className="AuthorizedURLForm"
+        >
+            <Field name="url">
+                <LemonInput
+                    defaultValue={urlToEdit}
```

Comment:
I think this may cause issues if passed together with a `value` component like this automatically is.

Reply:
Also, feel free to just use an array field of urls, and create unique fields with `<Field name={['urls', index]}>`